neurom/analysis/morphtree.py

```diff
@@ -44,8 +44,13 @@
 
 def path_length(tree):
```

Comment:
Recursion is a joke in python... Even with sections you will have to put a guard decorator or sth similar to check if you reached the extremely low recursion limit, in order to extend it, which sucks... -.- If you find however a nice way to do fast recursion, let me know. I would love to try it. :P